2 emerald test lab

Bartman250 opened the pull request:
More detail added to lab, gitignore added to prevent mac .DS_Store files

trbs commented:
@Bartman250 could you please add the role of `Mark Wagner` ?

Otherwise LGTM

vipinsun commented:
A small correction suggested: This performance is measure”d” against the Emerald benchmark.
Mark Wagner: chair of performance and scale WG & TSC member

vipinsun commented:
Other than those, LGTM

Bartman250 commented:
Thanks Vipin - I've updated the doc to reflect those changes.

tkuhrt commented on labs/emerald-performance-test-harness.md:
```diff
@@ -0,0 +1,68 @@
+The overall scope is to have a ready made test harness which blockchain
+developers can use to test and tune their own payment solutions.
+
+## Integration with hyperledger projects.
```
Can we capitalize Hyperledger in this section?

Bartman250 replied:
thanks Tracy - I've updated the doc

tkuhrt commented on labs/emerald-performance-test-harness.md:
```diff
@@ -0,0 +1,68 @@
+
+An example test harness has been written in javascript that currently 
+calls a Corda implementation of the payments network.
+The goal of this lab is to generacise this code so that it will
```
`generacise` -> `generalize`

Bartman250 replied:
thanks Tracy - I've updated the doc

tkuhrt commented on labs/emerald-performance-test-harness.md:
```diff
@@ -0,0 +1,68 @@
+An example test harness has been written in javascript that currently 
+calls a Corda implementation of the payments network.
+The goal of this lab is to generacise this code so that it will
+work with all of the hyperledger blockchain platforms in the hyperledger
```
`hyperledger`-> `Hyperledger` x2

Bartman250 replied:
thanks Tracy - I've updated the doc

tkuhrt commented on labs/emerald-performance-test-harness.md:
```diff
@@ -0,0 +1,68 @@
+calls a Corda implementation of the payments network.
+The goal of this lab is to generacise this code so that it will
+work with all of the hyperledger blockchain platforms in the hyperledger
+project and if appropriate plug into the caliper test harness.
```
`caliper` -> `Caliper`

Bartman250 replied:
thanks Tracy - I've updated the doc

trbs commented on labs/emerald-performance-test-harness.md:
```diff
@@ -0,0 +1,68 @@
+# Lab Name
+Performance Test Harness for Emerald Paymentd Benchmark. 
```
Should be "Payments" ?

Bartman250 commented:
@trbs thanks for the comments - I've renamed the lab to 'Payments Performance Test Harness'.
Corrected the spelling. and also the '2' in the PR title is just a number of the branch - nothing special

vipinsun commented:
lgtm+1

tkuhrt commented:
Looks like we have two +1s on this lab. I will create the lab as payments-performance-test-harness.

Bartman250 commented:
@tkuhrt Thanks for merging Tracy
